components/Navbar.tsx

```diff
@@ -115,7 +115,7 @@ export const Navbar: React.VFC = ({}) => {
                       </p>
                     </div>
                     <div className="py-1">
-                      <Link href="/account">
+                      <Link href="/account/accountPreferences">
```

Comment:
Let's simplify a bit the path i.e. `/account/settings`

Reply:
it has that name because the user goes to a page where he can change his email and/or password. I can rename it for sure but I would have to rename also the component and "settings" may be a bit confusing for this page. What should I do ?